apps/marketplace/components/BuyerDashboard/BuyerDashboardMyBriefs.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import ClosedDate from 'shared/ClosedDate'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import { loadBuyerDashboard } from 'marketplace/actions/dashboardActions'
+import { BUYER_DASHBOARD_MYBRIEFS_SUCCESS } from 'marketplace/constants/constants'
+import { statusConvert } from 'marketplace/components/helpers'
+import { rootPath } from 'marketplace/routes'
+import BuyerDashboardHelp from './BuyerDashboardHelp'
+import styles from './BuyerDashboard.scss'
+
+export class BuyerDashboardMyBriefs extends Component {
```

Comment:
again this probably should be SFC - define functions and state at the Page level and pass as props

Reply:
Makes these components easier to test as well